initializr-generator-spring/src/main/java/io/spring/initializr/generator/spring/code/kotlin/KotlinJpaGradleBuildCustomizer.java

```diff
@@ -46,7 +47,29 @@ public void customize(GradleBuild build) {
 		if (this.buildMetadataResolver.hasFacet(build, "jpa")) {
 			build.plugins()
 				.add("org.jetbrains.kotlin.plugin.jpa", (plugin) -> plugin.setVersion(this.settings.getVersion()));
+			if (this.buildMetadataResolver.hasGroupId(build, "jakarta.persistence")) {
+				customizeAllOpenWithJakarta(build);
+			}
+			else if (this.buildMetadataResolver.hasGroupId(build, "javax.persistence")) {
```

Comment:
We don't need to support JavaEE anymore. Our baseline is Spring Boot 3.2, which already uses JakartaEE.